.travis.yml

```diff
@@ -3,36 +3,39 @@ sudo: false
 
 env:
   global:
-    - OFFICIAL_REPO="slaclab/HXRSnD"    
+    - OFFICIAL_REPO="pcdshub/hxrsnd"    
     - secure: "ii21n3AE/32LdbxqP3r5sBlbxdHdZMBGk1icdDxsfinSNsXOYipCswMI3POZlSxHsO0g6uFtm5Qpi6WuNPknTrl7LqdpYDgnhhvneCyrEYMvOtK3weQLUqCjyPN0xx4KtzhAMXjNWAFccejQ16Z+0n4cU/4RU2mDZ/KfVPWNfXba7y/RGEVzGa8tCtJIHkSYZzIHS6ffWO1a6RsiBhDp0DTaV9pf20bnmCRzlrW8RlJ8c/5HXZ5mB11sMcM8S86XpElh6Wvts8cNq47Hv1u+BJg/vxJZJzkS8lGoXjsiu3k6vaZmmKV9Ylfgib1nEHZWbSpMAACaJ8X+7ltiBguTz6x7GnXVFx9AEn9J6tDzznCYs0xbs5ZMDxG6/yQWu4r8wF5Tk9rWwhTEBEdvCp+D+Sj2eUVeLkmB53FwPvIvr8ufe/vWSKHU3WNzTgchg3kt8Kq3WDgs+tERcYDZ/G7IJiDS1l2X8RyktP/Fa5fMgi6Uk6dEdFvTeI9ps/u/XaIjZz3kZREFNe/rBdOW8gOiki77UsnVapz9vPwu5cXI9Ap+WwvccvGzKMCKc7BvVnMkddslE5E6jf7LYNhWEs5YtSfNxuufSPg4d2oiqQEW0sBDhPnyGQ1mipbEfOcp+gMYrhQvUQfzZsmRJJdlz8uE8/tURFHJe7iyTyMHpqhVl8E="
 
 matrix:
-   include: 
-      - python: 3.5
-      - python: 3.6
-        env: BUILD_DOCS=1
+  include: 
+    - python: 3.6
+      env:
+        - PCDS_CHANNEL=pcds-tag
+    - python: 3.6
+      env:
+        - PCDS_CHANNEL=pcds-dev
+        - BUILD_DOCS=1
 
 install:
   - sudo apt-get update
-  # We do this conditionally because it saves us some downloading if the
-  # version is the same.
   - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
   - bash miniconda.sh -b -p $HOME/miniconda
   - export PATH="$HOME/miniconda/bin:$PATH"
   - hash -r
   - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
+  - conda install conda-build anaconda-client
+  - conda update -q conda conda-build
+  - conda config --add channels pcds-tag
+  - conda config --add channels $PCDS_CHANNEL
```

Comment:
Yeah, `--add` puts it at the start and `--append` puts it at the end. This is a copy paste of one of the other module's travis lines.

Reply:
That would make sense. The `append` syntax would be a little clearer but not worth changing here.

Reply:
I agree that append is more clear. Unfortunately, this means that our channels appear after defaults instead of before... We should look at the cleanest way to set these up so we don't confuse ourselves later.